File: frontend/lib/features/caja/models/nuevo_movimiento_dto.dart
class NuevoMovimientoDTO {
  final String tipo;
  final double monto;
  final String descripcion;
  final int usuarioId;
  final String tienda;

  NuevoMovimientoDTO({
    required this.tipo,
    required this.monto,
    required this.descripcion,
    required this.usuarioId,
    required this.tienda,
  });

  Map<String, dynamic> toJson() {
    return {
      'tipo': tipo,
      'monto': monto,
      'descripcion': descripcion,
      'usuarioId': usuarioId,
      'tienda': tienda,
    };
  }
}

File: Backend/src/main/java/com/optica/api/dto/NuevoMovimientoDTO.java
package com.optica.api.dto;

import com.optica.api.models.enums.Tienda;
import com.optica.api.models.enums.TipoMovimiento;
import lombok.Data;
import java.math.BigDecimal;

@Data
public class NuevoMovimientoDTO {
    private TipoMovimiento tipo; // ENTRADA o SALIDA
    private BigDecimal monto;
    private String descripcion;
    private Long usuarioId; // Quién hace el registro
    private Tienda tienda;
}